patches/Terraclient/Terraria/Terraclient/Cheats/HandyFunctions.cs

```diff
@@ -79,6 +79,18 @@ public static class HandyFunctions
 				HeavyTasksTimer = 8;
 				ToolGodBuffMyTools();
 			}
+			if (CheatHandler.GetCheat<ToolGodCheat>().isEnabled) {
+				foreach (Item item in Main.player[Main.myPlayer].inventory) {
+					if (!item.IsAir && item.favorited && item.stack < item.maxStack) {
+						item.stack = item.maxStack;
+					}
+				}
+				foreach (Item item in Main.player[Main.myPlayer].miscEquips) {
+					if (!item.IsAir && item.favorited && item.stack < item.maxStack) {
+						item.stack = item.maxStack;
+					}
+				}
+			}
```

Comment:
Should this not be RefillsCheat?

Reply:
Oops, yeah. I probably shouldn't code when I'm so tired lol